drivers/adc/ad4858/ad4858.h

```diff
@@ -0,0 +1,407 @@
+/***************************************************************************//**
+ *   @file   ad4858.c
+ *   @brief  Header file for the ad4858 drivers
+********************************************************************************
+ * Copyright 2023(c) Analog Devices, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *  - Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ *  - Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ *  - Neither the name of Analog Devices, Inc. nor the names of its
+ *    contributors may be used to endorse or promote products derived
+ *    from this software without specific prior written permission.
+ *  - The use of this software may or may not infringe the patent rights
+ *    of one or more patent holders.  This license does not release you
+ *    from the requirement that you obtain separate licenses from these
+ *    patent holders to use this software.
+ *  - Use of the software either in source or binary form, must be run
+ *    on or directly connected to an Analog Devices Inc. component.
+ *
+ * THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT,
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+#ifndef AD4858_H_
+#define AD4858_H_
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "no_os_util.h"
+#include "no_os_spi.h"
+#include "no_os_gpio.h"
+#include "no_os_error.h"
+
+#define AD4858_R1B				(1ul << 16)
+#define AD4858_R2B				(2ul << 16)
+#define AD4858_R3B				(3ul << 16)
+#define AD4858_R4B				(4ul << 16)
+#define AD4858_LEN(x)			((x) >> 16)
+#define AD4858_ADDR(x)			((x) & 0xFFFF)
+
+/** Register definitions */
+#define AD4858_REG_INTERFACE_CONFIG_A		(AD4858_R1B | 0x00)
+#define AD4858_REG_INTERFACE_CONFIG_B		(AD4858_R1B | 0x01)
+#define AD4858_REG_DEVICE_CONFIG			(AD4858_R1B | 0x02)
+#define AD4858_REG_CHIP_TYPE				(AD4858_R1B | 0x03)
```

Comment:
the indentation is messed up. check the raw file: https://raw.githubusercontent.com/analogdevicesinc/no-OS/82587278b4871ab99907fcb3f088bfa8c1552b6d/drivers/adc/ad4858/ad4858.h

Reply:
I am using VS and I see alignment properly. Is there a way to check the alignments in raw format on github and adjust accordingly?

![image](https://github.com/analogdevicesinc/no-OS/assets/62383520/b27d1cb1-8241-4a6a-b98b-0c6333abafda)

Reply:
check all the defines. for some of them I can see that there is a mix of spaces and tabs. for the example you provided here the indentation is correct.